packages/react/toast/src/Toast.stories.tsx

```diff
@@ -195,6 +195,49 @@ export const Animated = () => {
   );
 };
 
+export const Cypress = () => {
+  const [count, setCount] = React.useState(0);
+
+  return (
+    <Toast.Provider>
+      <button onClick={() => setCount((count) => count + 1)}>Add toast</button>
+
+      <div
+        style={{ display: 'flex', justifyContent: 'space-between', maxWidth: 700, margin: 'auto' }}
+      >
+        <button>Focusable before viewport</button>
+
+        {[...Array(count)].map((_, index) => {
+          const identifier = index + 1;
+          return (
+            <Toast.Root key={index} open className={rootClass()} data-cy={`toast-${identifier}`}>
+              <Toast.Title className={titleClass()}>Toast {identifier} title</Toast.Title>
+              <Toast.Description className={descriptionClass()}>
+                Toast {identifier} description
+              </Toast.Description>
+
+              <Toast.Close className={buttonClass()} aria-label="Close">
```

Comment:
We'd have to update the tests, but reading the tests it was weird to me why the close appeared before the action. Is there a particular reason? It made me question whether we were doing some reordering there too.